src/utils/const.ts

```diff
@@ -142,11 +165,14 @@ export const SPARK_HISTORY_SERVER = 'Spark History Server';
 export const DEFAULT_LABEL_DETAIL = 'client:dataproc-jupyter-plugin';
 export const JOB_FIELDS_EXCLUDED = ['queryList', 'properties', 'args'];
 export const BATCH_FIELDS_EXCLUDED = ['queryList', 'properties'];
-export const GCS_URL = 'https://storage.googleapis.com/storage/v1/b';
-export const GCS_UPLOAD_URL = 'https://storage.googleapis.com/upload/storage/v1/b'
-export const KEY_MESSAGE='Example format:projects/<project-name>/locations/<location-name>/keyRings/<keyring-name>/cryptoKeys/<key-name>';
+export const GCS_URL = storedUrlResponseObj.storage_url + 'storage/v1/b';
```

Comment:
does this work?  It looks like in urlhandler we have:
`storage_url = gcp_service_url('storage', default_url='https://storage.googleapis.com/storage/v1/')`

Wouln't that mean that GCS_URL would be https://storage.googleapis.com/storage/v1/storage/v1/b

in the default case?